pages/_data/glossary.json

```diff
@@ -0,0 +1,222 @@
+[
```

Comment:
Just a suggestion, but can we implement glossary terms as a collection instead of raw JSON? It's easier for content editors to work with yaml/markdown over JSON.

There's an example from [www.foia.gov](https://github.com/usdoj/foia.gov/tree/develop/www.foia.gov/_glossary_terms).

Reply:
Agree! We didn't have time to revisit this glossary CSV setup, Ideally we'd have the glossary as a proper collection -- currently the `page_gen` plugin processes this CSV and transforms it into a collection for us. Most of this is taken from PRA -- thanks for the FOIA example!